<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server</class>
 <widget class="QWidget" name="server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>本地ip地址：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxIp">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>监听端口号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort">
       <property name="text">
        <string>12345</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="buttonListen">
     <property name="text">
      <string>开始监听</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="messageShow"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditSend"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListView" name="listViewConnectShow"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
